taxonomies/categories.txt

```diff
@@ -22603,10 +22611,6 @@ de:Schmand mit 15% Fettanteil
 fr:Smetana avec 15% de matières grasses
 ru:Сметана 15%, Сметана 15%, Сметана 15% жирности, Сметана с массовой долей жира 15%
 
-<en:Creams
```

Comment:
Are you removing the en:Whipped creams entry intentionaly?

Reply:
Yes, it was in double in the taxonomy. However I haven't seen that the wikidata codes were different. The good one is Q625957.